src/ArmTemplates/Common/Templates/Policy/PolicyTemplateProperties.cs

```diff
@@ -4,11 +4,14 @@
 //  </copyright>
 // --------------------------------------------------------------------------
 
+using Newtonsoft.Json;
+
 namespace Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common.Templates.Policy
 {
     public class PolicyTemplateProperties
     {
-        public string Value { get; set; }
+        [JsonProperty("value")]
```

Comment:
Shouldn't this be "Value"?

Reply:
answered [here](https://github.com/Azure/azure-api-management-devops-resource-kit/pull/639#discussion_r823682497)